mitodl/hugo-course-publisher
# Update sample markdown

#### Pre-Flight checklist


#### What are the relevant tickets?
https://trello.com/c/HQzBhlkR/49-fix-course-image-on-the-desktop

#### What's this PR do?
Added updated markdown based on latest json in open-learning-course-data and changes to ocw-to-hugo to support course image alt and caption text.

#### How should this be manually tested?
Run the site and verify that everything is built correctly without errors and none of the content is changed or messed up.
